### ISSUE TITLE
[<package-id>] mcdonalds app requests play store update (huawei p60 pro) 

### ISSUE THREAD

**Issue by ilias-vs** [microg/GmsCore#2162]
**McDonald's app**
Name: [McDonald's app]
Package id: [com.mcdonalds.mobileapp]

**To Reproduce* 
When the app starts and choose language after I press okay there is an error. The error says "it seems like your version of the play store is out of date. To use McDonald's app on this device update playstore to the latest version" 

**Expected behavior**
![Screenshot_20240129_124933_com.mcdonalds.mobileapp.jpg](https://github.com/microg/GmsCore/assets/102998139/096ebc69-8012-46c2-ab28-aa5a2c1c17ad)

**System**
Android Version: [emui 13.1.0]
Custom ROM: [harmonyOS 3]

**Comment by ale5000-git** [comment]
Increasing version of FakeStore doesn't seem to help.
I'm not sure but maybe it is related to this:
`01-29 21:49:22.845  2386  4723 W ActivityManager: Unable to start service Intent { act=com.google.android.play.core.integrityservice.BIND_INTEGRITY_SERVICE pkg=com.android.vending } U=0: not found`

**Comment by ilias-vs** [comment]
So i need some kind of spoofing signature to bypass that problem. I have been searching online and i need to root my device and add some spoofing. Is there some other way without rooting my device

**Comment by Wojtaz0w** [comment]
You're incorrect. It requires support for Play Integrity API. It *literally* says that in log 🤦🏻‍♂️

**Comment by Wojtaz0w** [comment]
As far as I know to allow apps to check system via this API you need original, not modified Play store, not FakeStore. Also it will probably fail without Play Integrity Fix module for Magisk/KSU.

**Comment by ilias-vs** [comment]
Thank you for the response i ll try to work around that

**Comment by Wojtaz0w** [comment]
Hi friend @ilias-vs , any update? If so let me know. I'll try to help if you'll need it. Have a good day

**Comment by ale5000-git** [comment]
The pull request to add the support is here: https://github.com/microg/GmsCore/pull/2077
but I don't think anyone is working on it currently.

**Comment by ilias-vs** [comment]
> Hi friend @ilias-vs , any update? If so let me know. I'll try to help if you'll need it. Have a good day
> 

Hello, not any luck yet,for now a lot of googling. If you have any tips or guides i ll appreciate them
